two-tier-app-migration-to-containers-on-Azure/nested/webDeploy.json

```diff
@@ -0,0 +1,64 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "virtualMachineName": {
+      "type": "string",
+      "minLength": 1
+    },
+    "sqlAuthenticationPassword": {
+         "type": "securestring",
+         "metadata": {
+             "description": "Password for the database user dbuser."
+                }
+       },
+    "_artifactsLocation": {
+      "type": "string",
+      "metadata": {
+        "description": "The base URI where artifacts required by this template are located. When the template is deployed using the accompanying scripts, a private location in the subscription will be used and this value will be automatically generated."
+      },
+      "defaultValue": "https://raw.githubusercontent.com/SpektraSystems/two-tier-app-migration-containers/master/"
+    },
+    "_artifactsLocationSasToken": {
+      "type": "securestring",
+      "metadata": {
+        "description": "The sasToken required to access _artifactsLocation.  When the template is deployed using the accompanying scripts, a sasToken will be automatically generated."
+      },
+      "defaultValue": ""
+    }
+  },
+  "variables": {
+    "scriptFolder": "scripts/",
+    "scriptFileName": "envcreate.ps1",
+    "dbcreateScript": "[concat(parameters('_artifactsLocation'), variables('scriptFolder'), variables('scriptFileName'), parameters('_artifactsLocationSasToken'))]",
+    "url1": "[concat(parameters('_artifactsLocation'), 'scripts/DotNetAppSqlDb.deploy.cmd', parameters('_artifactsLocationSasToken'))]",
+    "url2": "[concat(parameters('_artifactsLocation'), 'scripts/DotNetAppSqlDb.SetParameters.xml', parameters('_artifactsLocationSasToken'))]",
+    "url3": "[concat(parameters('_artifactsLocation'), 'scripts/DotNetAppSqlDb.zip', parameters('_artifactsLocationSasToken'))]",

```

Comment:
this needs to come from a secure location, we can't check compiled binaries into the sample (they can't be verified)